docs/release-testing.md

```diff
@@ -1,137 +1,70 @@
 
-Deploy the catalog source to the cluster:
-
-```sh
-$ oc apply -f catalog-source.yaml
-```
-
-Verify that the catalog source was created without errors in the `openshift-marketplace` project.
-
-Search OperatorHub for the koku-metrics-operator. It should be available under the `custom` or `my-test-catalog` provider type depending on your OCP version.
-
-Install the koku-metrics-operator in the `koku-metrics-operator` namespace, and test as normal.
-
-
-3. Generate the new controller image, and push to Quay:
+2. Generate the new controller image, and push to your Quay repo:
 
 ```sh
 $ USERNAME=<quay-username>
 $ VERSION=<release-version>
-$ docker build . -t quay.io/$USERNAME/koku-metrics-operator:v$VERSION
-$ docker push quay.io/$USERNAME/koku-metrics-operator:v$VERSION
+$ make docker-buildx IMG=quay.io/$USERNAME/koku-metrics-operator:v$VERSION
 ```
 
 4. Generate the new bundle:
 
-Update the release versions at the top of the `Makefile` to match the release version of the operator:
-
-```
-# Current Operator version
-PREVIOUS_VERSION ?= <version to upgrade from>
-VERSION ?= <release-version>
-```
-
 Run the following command to generate the bundle:
 
 ```sh
+$ docker pull quay.io/$USERNAME/koku-metrics-operator:v$VERSION
 $ make bundle CHANNELS=alpha,beta DEFAULT_CHANNEL=beta IMG=quay.io/$USERNAME/koku-metrics-operator:v$VERSION
 ```
 
-This will generate a new `<release-version>` bundle inside of the `koku-metrics-operator` directory within the repository.
-
-5. Build the bundle image and push to Quay:
+This will generate a new `$VERSION` bundle inside of the `koku-metrics-operator` directory within the repository.
 
-Copy the generated bundle to the testing directory:
-
-```sh
-$ mv koku-metrics-operator/$VERSION testing
-$ cd testing/$VERSION
-```
+1. Build the bundle image and push to Quay:
 
 Build and push bundle to your Quay repo:
 
 ```sh
-$ docker build -f Dockerfile . -t quay.io/$USERNAME/koku-metrics-operator-bundle:v$VERSION; docker push quay.io/$USERNAME/koku-metrics-operator-bundle:v$VERSION
+$ make bundle-build BUNDLE_IMG=quay.io/$USERNAME/koku-metrics-operator-bundle:v$VERSION bundle-push
 ```
 
-6. Create an opm index (test-catalog) that contains the last release and the new release, and push to Quay:
-
+1. Once the bundle is available in Quay, deploy it to your cluster:
 ```sh
-$ opm index add --from-index quay.io/project-koku/kmc-test-catalog:v$PREVIOUS_VERSION --bundles quay.io/$USERNAME/koku-metrics-operator-bundle:v$VERSION --tag quay.io/$USERNAME/test-catalog:v$VERSION --container-tool docker
-$ docker push quay.io/$USERNAME/test-catalog:v$VERSION
+$ make bundle-deploy-upgrade BUNDLE_IMG=quay.io/$USERNAME/koku-metrics-operator-bundle:v$VERSION
 ```
 
-7. Update the image in the CatalogSource to point to the new test-catalog version (i.e. `image: quay.io/$USERNAME/test-catalog:v$VERSION`).
 8. Observe the installed operator, and ensure it upgrades automatically.
```

Comment:
Sam thing here. The contents of each step need to be indented so that the step order is calculated correctly.

```markdown
1. Set version numbers in the Makefile:

    ```sh
    $ PREVIOUS_VERSION=0.9.8
    $ VERSION=0.9.9
    ```

1. Create the namespace in the cluster and deploy the previous bundle:

    ```sh
    $ oc new-project koku-metrics-operator
    $ make bundle-deploy-previous
    ```

    Check the `koku-metrics-operator` namespace and ensure the previous version deployed correctly. Create a KokuMetricsConfig so that the PVC is created and data is collected, if available.

1. Generate the new controller image, and push to your Quay repo:

    ```sh
    $ USERNAME=<quay-username>
    $ VERSION=<release-version>
    $ make docker-buildx IMG=quay.io/$USERNAME/koku-metrics-operator:v$VERSION
    ```

1. Generate the new bundle:

    Run the following command to generate the bundle:

    ```sh
    $ docker pull quay.io/$USERNAME/koku-metrics-operator:v$VERSION
    $ make bundle CHANNELS=alpha,beta DEFAULT_CHANNEL=beta IMG=quay.io/$USERNAME/koku-metrics-operator:v$VERSION
    ```

    This will generate a new `$VERSION` bundle inside of the `koku-metrics-operator` directory within the repository.

1. Build the bundle image and push to Quay:

    Build and push bundle to your Quay repo:

    ```sh
    $ make bundle-build BUNDLE_IMG=quay.io/$USERNAME/koku-metrics-operator-bundle:v$VERSION bundle-push
    ```

1. Once the bundle is available in Quay, deploy it to your cluster:

    ```sh
    $ make bundle-deploy-upgrade BUNDLE_IMG=quay.io/$USERNAME/koku-metrics-operator-bundle:v$VERSION
    ```

1. Observe the installed operator, and ensure it upgrades automatically.

    Once the operator has upgraded, run through manual tests as normal.
```